meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -0,0 +1,2 @@
+# XXX Drop the VM hack after taking care also of the non UEFI devices (those using U-Boot: edison and beaglebone)
+OSTRO_VM_IMAGE_TYPES = "dski.zip dsk.vdi.zip"
```

Comment:
Typo: s/dski.zip/dsk.zip/

To address kad's earlier concerns for the previous patches, overriding OSTRO_VM_IMAGE_TYPES unconditionally here is fine, because these are exactly the values the CI is to use, i.e. nothing needs to change them.

@kad How do you want the CI to use include files? Include both "ostro-development.inc" and "ostroproject-ci.inc", or just "ostroproject-ci.inc"? I had the impression that it is the latter, in which case this file here must include ostro-development.inc.